### PR TITLE
add ability to select an image size on hero widget

### PR DESCRIPTION
**Pull request by rlamarche**
This PR add a form field to select the format of image in the background of each frame.
It is backward compatible (default value is "Full").

### CODE REVIEW

File: widgets/hero/hero.php [hunk 1]
```diff
@@ -261,7 +266,7 @@ function get_widget_form(){
 	function get_frame_background( $i, $frame ){
 		$background_image = siteorigin_widgets_get_attachment_image_src(
 			$frame['background']['image'],
-			'full',
+			$frame['background']['size'],
```
Comment (braamgenis) [inline widgets/hero/hero.php]: This needs an `empty` check and should fall back to using `'full'` for existing widget instances which won't yet have the `$frame['background']['size']` property.

**Comment by rlamarche** [comment]
You're welcome @braamgenis  :) and thank you for this wonderful wordpress extension, very easy to improve :+1: 
I just made the change you asked. I thought it was an automatic behaviour because I tested the change on an existing widget and I had no problems, but anyway it's checked now ;)

**Comment by braamgenis** [comment]
@rlamarche Yeah, it does "work" with existing instances, but the `size` field is an empty string which may have unintended consequences for some, so better to make completely sure the default is what we expect it to be. It's better to be careful when you have such a large number of active installations. :wink:

**Comment by rlamarche** [comment]
Ok I fully agree with you, and thank you for the merge :+1:

**Pull request merged**